.aicoe-ci.yaml

```diff
@@ -0,0 +1,16 @@
+check:
+  - thoth-build
+  - thoth-precommit
+  - thoth-pytest-py38
+build:
+  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.20.1
+  build-stratergy: Source
+  registry: quay.io
+  registry-org: thoth-station
+  registry-project: invectio
+  registry-secret: thoth-station-thoth-pusher-secret
+deploy:
+  project-org: "thoth-station"
+  project-name: "thoth-application"
+  image-name: "invectio"
+  overlay-contextpath: "invectio/overlays/test/imagestreamtag.yaml"
```

Comment:
This is not needed as invectio is a python package module, not a application component.